packit_service_fedmsg/consumer.py

```diff
@@ -33,6 +33,7 @@
 INTERESTED_TOPICS = {
     "org.fedoraproject.prod.copr.build.end",
     "org.fedoraproject.prod.copr.build.start",
+    "org.fedoraproject.prod.buildsys.task.state.change"
```

Comment:
This `INTERESTED_TOPICS` was actually [added](https://github.com/packit-service/packit-service-fedmsg/commit/98fea0474de964314ceb89a3ae299cc6e57bd5c6) back in the days when we were seeing some infrastructure problems related to `copr` & `fedora-messaging` - and this was one of the workarounds.

I think it can actually be removed now because we haven't seen any problems for quite some time.

I can accept/merge this PR now and create a separate one if you want.

The place where it needs to be added is https://gitlab.cee.redhat.com/user-cont/packit-service/merge_requests/85

Reply:
Thanks for creating the MR.

I'll remove it and rebase once that PR is merged.